src/client/Microsoft.Identity.Client/Internal/Broker/BrokerSilentRequest.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.UI;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Internal.Broker
+{
+    internal class BrokerSilentRequest
+    {
+        internal Dictionary<string, string> BrokerPayload { get; set; } = new Dictionary<string, string>();
+        internal IBroker Broker { get; }
+        private readonly AcquireTokenSilentParameters _silentParameters;
+        private readonly AuthenticationRequestParameters _authenticationRequestParameters;
+        private readonly IServiceBundle _serviceBundle;
+        //private readonly AuthorizationResult _authorizationResult;
```

Comment:
I assume there is no call to the /authorize endpoint and no AuthroizationResult needed...